### PR TITLE
Rename id field to uuid in Identity

### PR DESCRIPTION
**Pull request by sduenas**
This PR renames the field `id` to `uuid` in `Identity` model. UUID will be the identifier of an `Identity`.

This PR is related to #290.

### CODE REVIEW

File: tests/test_api.py [hunk 1]
```diff
@@ -1525,7 +1524,7 @@ def test_from_id_is_individual(self):
         trx_date = datetime_utcnow()  # After this datetime no transactions should be created
 
         # Check 'from_id' parameter
```
Comment (mafesan) [inline tests/test_api.py:1526]: I think the name and the description of this test should be updated to `test_from_uuid_is_individual`. This line comment should be updated too with the new parameter name.

File: tests/test_api.py [hunk 2]
```diff
@@ -1587,7 +1586,7 @@ def test_empty_from_id(self):
 
         trx_date = datetime_utcnow()  # After this datetime no transactions should be created
 
-        with self.assertRaisesRegex(InvalidValueError, FROM_ID_NONE_OR_EMPTY_ERROR):
+        with self.assertRaisesRegex(InvalidValueError, FROM_UUID_NONE_OR_EMPTY_ERROR):
```
Comment (mafesan) [inline tests/test_api.py:1589]: Same as above about the name of this test and its description.

**Comment by sduenas** [comment]
@mafesan please check the changes. I updated all the references to `from_id` I found.

**Comment by mafesan** [comment]
Hi @sduenas, I have reviewed the changes and I still see some, please check:

[L1570](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1570) (optional), [L1571](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1571), [L1629](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1629),  [L1638](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1638),  [L1645](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1645),  [L1648](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1648), [L1664](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1664),  [L1667](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1667), [L1679](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1679) and [L1685](https://github.com/chaoss/grimoirelab-sortinghat/pull/291/files#diff-625efbbf35a2290861f596fee1b19838R1685).

**Comment by sduenas** [comment]
> Hi @sduenas, I have reviewed the changes and I still see some, please check:
> 

Sorry @mafesan I thought I commited all the changes. Please check again.

**Pull request merged**